app/src/main/java/org/fossasia/openevent/Adapters/TracksListAdapter.java

```diff
@@ -44,6 +45,14 @@ public int getItemCount() {
         return tracks.size();
     }
 
+    public void deleteItems() {
+        Log.d("delete", "tracks");
+        for (int i = 0; i < getItemCount(); i++) {
+            Log.d("i" + i, tracks.get(i).getName());
+            tracks.remove(i);
+            notifyItemRemoved(i);
```

Comment:
I think should be done after you have finished removing? otherwise too many notify calls will cause too many getView.